walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -218,46 +139,47 @@ In this task, you use the Identity Management service to protect both the client
 
 === Add Identity Management service to the client app (PWA) and create a realm for users
 
-. Navigate to the link:{mdc-url}[Mobile Developer Console, window="_blank"].
-. Click on the *showcase-app* card to display the configuration screen.
-. Click the *Mobile Services* tab.
-. Click *Bind to App* for *Identity Management* in the *Unbound Services* section and then click the *next* button.
-.. When prompted for an *Admin User Name* enter:
-+
-----
-admin
-----
-.. When prompted for an *Admin User Password* enter:
-+
-----
-admin
-----
-. Complete the wizard.
-This process also creates a new realm in the {customer-sso-name}.
+. As your admin to create new realm for your user and obtain url to be able to login to the keycloak instance
+. In new realm we going to create two new clients - one for server side and one for the client side application.
```

Comment:
is this line part of the instruction? if not we can remove this.

Reply:
Also the steps from here doesn't really belong in this section: it's about getting the KC client config for the client app.

Reply:
Got it cleared based on my understanding but feel free to rebuild it